addons/account/models/account_move.py

```diff
@@ -361,7 +361,7 @@ def _get_counterpart(self):
             counterpart = list(counterpart)[0:2] + ["..."]
         self.counterpart = ",".join(counterpart)
 
-    name = fields.Char(required=True, string="Label")
+    name = fields.Char(string="Label")
```

Comment:
the idea is to simplify the manual encoding of journal entries: since you input a single operation, you rather want to write once its label (and will use the reference field of the move instead of copying/pasting the same label on all the account.move.line names